arbnode/api.go

```diff
@@ -59,19 +58,3 @@ func (a *BlockValidatorDebugAPI) ValidationInputsAt(ctx context.Context, msgNum
 ) (server_api.InputJSON, error) {
 	return a.val.ValidationInputsAt(ctx, arbutil.MessageIndex(msgNum), target)
 }
```

Comment:
we need an API that will return an error if maintenance is currently running. This will be used by SRE to e.g. make sure there are enough active node without counting those running maintenance.

The exact RPC doesn't have to be "SecondsSinceLastMaintenance", but it does have to return an error if currently running and no error otherwise.

Reply:
Possibly this could be an API that's directly connected to the execution side

Reply:
Added an arb_maintenanceStatus API on Execution.

My original idea was that the MaintenanceStatus API, in which Consensus polls Execution to check if maintenance was completed, would also be used by node operators, but this is not an RPC API yet :(.

arb_maintenanceStatus API doesn't return an error in case maintenance is not running though, returns a struct with a isRunning bool.